lib/bigcommerce/resources/customers/customer.rb

```diff
@@ -26,5 +28,21 @@ class Customer < Resource
     def self.count(params = {})
       get 'customers/count', params
     end
+
+    # Generate a token that can be used to log the customer into the storefront.
+    # This requires your app to have the store_v2_customers_login scope and to
+    # be installed in the store.
+    def login_token
+      payload = {
+        'iss' => Bigcommerce.config.client_id,
+        'iat' => Time.now.to_i,
+        'jti' => SecureRandom.hex(16),
```

Comment:
Not sure this is good enough, maybe client_id and a uuid?

https://self-issued.info/docs/draft-ietf-oauth-json-web-token.html#rfc.section.4.1.7